net/src/client/tcp/actors/jsonrpc.rs

```diff
@@ -58,6 +63,17 @@ impl JsonRpcClient {
 
     /// Replace the TCP connection with a fresh new connection.
     pub fn reconnect(&mut self, ctx: &mut <Self as Actor>::Context) {
+        let now = Instant::now();
+        // Attempt to reconnect at most one every 10 seconds
+        let reconnection_cooldown = Duration::from_secs(10);
```

Comment:
Just out of curiosity, can this be made into a `const`?

Reply:
Probably yes, I also considered using the default timeout of 5 seconds here.